ci/screenshot.Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM python:3.11-slim-buster
+FROM python:3.11-slim-bookworm
```

Comment:
I think the `gcc` version was too old for the newer google chrome stuff. This updates to a newer one